layouts/index.tsx

```diff
@@ -37,6 +42,9 @@ const MDX_COMPONENTS = {
   CardGrid,
   Card,
   code: InlineCode,
+  h1: AnchoredH1,
```

Comment:
For now I only have it providing anchor links for `h1..h3` to match the listing in On This Page, but we can always add more if we want.